src/window/editor/scenario_event_details.c

```diff
@@ -207,10 +232,12 @@ static void draw_foreground(void)
         SHORT_BUTTON_LEFT_PADDING + 16, y_offset + 8, FONT_NORMAL_PLAIN, COLOR_BLACK);
 
     y_offset = DETAILS_Y_OFFSET;
+    int i_button_offset = 4;
     for (int i = 0; i < MAX_VISIBLE_ROWS; i++) {
         if (data.list[i].sub_type != SUB_ITEM_TYPE_UNDEFINED) {
-            large_label_draw(buttons[i+3].x, buttons[i+3].y, buttons[i+3].width / 16, data.focus_button_id == i + 4 ? 1 : 0);
-            if (data.focus_button_id == (i + 4)) {
+            large_label_draw(buttons[i+i_button_offset].x, buttons[i+i_button_offset].y, buttons[i+i_button_offset].width / 16, data.focus_button_id == i + i_button_offset + 1 ? 1 : 0);
```

Comment:
```suggestion
            large_label_draw(buttons[i + i_button_offset].x, buttons[i + i_button_offset].y, buttons[i+i_button_offset].width / 16, data.focus_button_id == i + i_button_offset + 1 ? 1 : 0);
```

We always use a space between variables and `+`, even inside array items.

Reply:
Done.